src/nvim/ex_cmds.c

```diff
@@ -3182,6 +3201,11 @@ void do_sub(exarg_T *eap)
   /*
    * Check for a match on each line.
    */
+
+  //  list to save matched lines
+  //  Note: Could not use nolint with rule, line would be > 100 chars
+  klist_t(matchedline_T) *lmatch = kl_init(matchedline_T); // NOLINT
```

Comment:
Need to update the linter like in 458a4d04447e57c198210fb8f743a261e13b9855, not use NOLINT.